src/vivarium/framework/results/context.py

```diff
@@ -90,16 +91,16 @@ def add_observation(
         pop_filter: str,
         aggregator_sources: Optional[List[str]],
         aggregator: Callable[[pd.DataFrame], float],
-        additional_stratifications: List[str] = [],
```

Comment:
I removed defaults for a bunch of these functions that are ultimately wrapped in the ResultsInterface object so that I could tack `report` on the end without having to keep passing the default method through. I also think it's cleaner this way (though it does mean I had to fix some of the unit tests that were calling these inner methods)

Reply:
See if removing the defaults is really providing value as you can set a default report function, though I don't think there's anything wrong with doing it as you have.